spec/classes/php_spec.rb

```diff
@@ -50,6 +50,51 @@
         end
       end
 
+      describe 'when called with fpm_user parameter' do
+        let(:params) { { fpm_user: 'nginx' } }
+        it { is_expected.to contain_class('php::fpm').with(user: 'nginx') }
+        it { is_expected.to contain_php__fpm__pool('www').with(user: 'nginx') }
+        # rubocop:disable RSpec/RepeatedExample
```

Comment:
maybe you could instead do:

```
dstfile = case facts[:operatingsystem]
when 'Ubuntu'
  '/etc/php5/fpm/pool.d/www.conf'
```

it { is_expected.to contain_file(dstfile).with_content(/user = nginx/) }